src/app/api/getStakerPoints.ts

```diff
@@ -0,0 +1,29 @@
+import { encode } from "url-safe-base64";
+
+import { apiWrapper } from "./apiWrapper";
+
+export interface StakerPoint {
+  staker_btc_pk: string;
+  polkadots: number;
+}
+
+export const getStakerPoints = async (
+  stakerBtcPk: string | string[],
```

Comment:
Why don't we make this to only receive an array of strings to make our life easier and have a unified way to call it?